src/app/core/basic-datatypes/date/date-import-config/date-import-dialog.component.html

```
@@ -0,0 +1,48 @@
+    </mat-hint>
+  </mat-form-field>
+
+  <app-help-button
+    text="Define how date values in your imported data are formatted, so that the system correctly understands them. Values that do not match the given format will be ignored (remain empty) during import."
+    i18n-text="import - value mapping (date) - help text"
+  ></app-help-button>
+
+  <mat-list>
+    @for (val of values; track val) {
+      <mat-list-item [class.invalid]="!val.parsed">
+        {{ val.value }} -> {{ val.parsed | customDate }}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)